src/composables/trade/constants.ts

```diff
@@ -1 +0,0 @@
-export const ENABLE_LEGACY_TRADE_INTERFACE = false;
```

Comment:
Just delete this file.

Reply:
It's deleted :) It's Just how gitHub shows the diff.

Reply:
ahh ok, yeah my bad. Usually, the diff just says "This file was deleted" but it didn't for this one. I guess because the diff is just one line or something.